react/components/AdminMessages.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useState } from 'react'
+import { InjectedIntl, injectIntl } from 'react-intl'
+
+import { Layout, PageHeader } from 'vtex.styleguide'
+
+import { AddProductTranslationsMutationFn } from '../mutations/AddProductTranslations'
+import { MessagesOfProvider, StepCounterControl } from '../typings/typings'
+import ProductMessagesExport from './ProductMessagesExport'
+import ProductMessagesImport from './ProductMessagesImport'
+
+const TOTAL_STEPS = 3
+
+interface Props {
+  intl: InjectedIntl
+  addProductTranslations: AddProductTranslationsMutationFn
+  email: string
+}
+
+const AdminMessages: FC<Props> = ({ email, intl }) => {
+  const [step, setStep] = useState(1)
+
+  const translationDataHooks = {
+    language: useState(''),
+    messages: useState<MessagesOfProvider[] | null>(null),
+  }
+
+  const next = () => setStep(step + 1)
```

Comment:
[`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) 😝

Reply:
Since it is a simple closure that doesn't re render very often, I judged it was not necessary.